main.c

```diff
@@ -972,6 +972,8 @@ int main(int argc, char **argv)
 	int fd;
 	int ret;
 
+	tcmu_info("Starting...\n");
```

Comment:
Do you need this? Won't you get it from systemd too?

Reply:
Yeah, we need it, in some container case, there no syslog service or even the journalctl command.
What we can get will only from the tcmu-runner.log. And mostly as default there isn't any log output for tcmu-runner in that log file, it seems the logger is not working for tcmu-runner, and for this case has been raised one bug by our QE.